src/docs/product/discover-queries/index.mdx

```diff
@@ -110,7 +110,7 @@ Each table cell has a dynamic context menu that allows you to continue exploring
 
 ![Context menu](013.png)
 
-If the first column shows an icon (as seen above), it means the events have been stacked. Click on the icon to view the full list of events. Each event will have an event ID you can click on for more details. For more about how to build a query, go to [the Query Builder](/product/discover-queries/query-builder/).
+If the first column shows an icon (as seen above), it means the events have been stacked. Click on the icon to view the full list of events. Each event will have an event ID you can click on for more details. Hovering over fields under the issue,release and event id columns displays additional information like issue statuses, release authors and stack traces for errors. For more about how to build a query, go to [the Query Builder](/product/discover-queries/query-builder/).
```

Comment:
I moved the last sentence down just because it doesn't fit as well with the rest of the paragraph now that we've added more. Formatted the UI elements and added Oxford/serial commas.
```suggestion
If the first column shows an icon (as seen above), it means the events have been stacked. Click on the icon to view the full list of events. Each event will have an event ID you can click on for more details. Hovering over fields in the "Issue", "Release", and "Event ID" columns displays additional information like issue statuses, release authors, and stack traces for errors. 

For more about how to build a query, go to [the Query Builder](/product/discover-queries/query-builder/).
```